GameDefinition/Content/ItemKindActor.hs

```diff
@@ -438,7 +437,7 @@ beeSwarm = ItemKind
                , AddAbility AbAlter (-2) ]  -- can't use stairs nor doors
   , ieffects = []
   , ifeature = [Durable, Identified]
-  , idesc    = ""
+  , idesc    = "Every bee would die for the queen."
```

Comment:
OK, let's keep it as is. Cryptic is fine, because it alerts to possibly unobvious behaviour and surprises are a good thing and the healing effect is rather apparent after the first sting.